src/marqo/core/unstructured_vespa_index/unstructured_vespa_index.py

```diff
@@ -16,7 +16,7 @@
 class UnstructuredVespaIndex(VespaIndex):
     _FILTER_STRING_BOOL_VALUES = ["true", "false"]
     _RESERVED_FIELD_SUBSTRING = "::"
-    _SUPPORTED_FIELD_CONTENT_TYPES = [str, int, float, bool, list]
+    _SUPPORTED_FIELD_CONTENT_TYPES = [str, int, float, bool, list, dict]
```

Comment:
Why do we need dict here?

Reply:
Because field content for `custom_vector` is now a `dict`. It will have 2 keys (`content` and `vector`)

Reply:
What happens if we pass dict but it's not a custom_vector field?